### PR TITLE
Added English and German translation files.

### PR DESCRIPTION
**Pull request by emanuelschuetze**
- Use gulp and bower to create client-side translations with angular-gettext.
- Add English po template files for django and angular-gettext.
- Added German translation files.
- Fixed some English strings.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,8 +7,13 @@ __pycache__/
 .virtualenv/
 .venv/
 
-# Local user data (settings, database, media, search index, static files)
```
Comment (normanjaeckel) [inline .gitignore:10]: Please readd these lines. They are required for Plugin development with virtualenv installed OpenSlides (core).

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "openslides-votecollector-plugin",
```
Comment (normanjaeckel) [inline bower.json]: I would prefer `openslides-votecollector`.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "openslides-votecollector-plugin",
```
Comment (normanjaeckel) [inline package.json]: Here the same.

File: .gitignore (hunk 1 above)
Comment (emanuelschuetze) [reply .gitignore:10]: done

File: bower.json (hunk 2 above)
Comment (emanuelschuetze) [reply bower.json]: done

File: package.json (hunk 3 above)
Comment (emanuelschuetze) [reply package.json]: done

**Comment by emanuelschuetze** [comment]
@normanjaeckel All done. Ready for review and  merge.

**Comment by emanuelschuetze** [comment]
@normanjaeckel Ping. Ready for review and merge.

**Comment by normanjaeckel** [comment]
Please strip out all *.mo files and add a pattern for them to the gulpignore file.

**Comment by emanuelschuetze** [comment]
Why? We have to commit the German mo file to use the django translations.

**Comment by emanuelschuetze** [comment]
Only the English mo file is optional (not for usage.)

File: openslides_votecollector/views.py [hunk 4]
```diff
@@ -114,8 +114,10 @@ def anonymize_votes(self, request):
         """
         Anonymize all votes of the given poll.
         """
-        MotionPollKeypadConnection.objects.filter(poll_id=request.data.get('poll_id')).update(keypad=None)
-        # TODO: Trigger autoupdate.
+        # Clear keypad id and save model to trigger autoupdate.
+        for apkc in MotionPollKeypadConnection.objects.filter(poll_id=request.data.get('poll_id')):
```
Comment (normanjaeckel) [inline openslides_votecollector/views.py]: typo: `mpck`

File: openslides_votecollector/views.py [hunk 5]
```diff
@@ -501,12 +533,12 @@ def post(self, request, item_id, keypad_id):
             except OpenSlidesError:
                 # User is already on the speaker list.
                 pass
-            content = _('Added to list of speakers')
+            content = _('Added to        list of speakers')
```
Comment (normanjaeckel) [inline openslides_votecollector/views.py:536]: Why did you add whitespaces?

File: openslides_votecollector/views.py [hunk 6]
```diff
@@ -501,12 +533,12 @@ def post(self, request, item_id, keypad_id):
...
         # Remove keypad user from the speaker list.
         elif value == 'N':
             # Remove speaker if on "next speakers" list (begin_time=None, end_time=None).
             Speaker.objects.filter(user=keypad.user, item=item, begin_time=None, end_time=None).delete()
-            content = _('Removed from list of speakers')
+            content = _('Removed from    list of speakers')
```
Comment (normanjaeckel) [inline openslides_votecollector/views.py:541]: Why did you add whitespaces?

**Comment by normanjaeckel** [comment]
@emanuelschuetze Some remarks.

File: openslides_votecollector/views.py (hunk 4 above)
Comment (emanuelschuetze) [reply openslides_votecollector/views.py]: done

File: openslides_votecollector/views.py (hunk 6 above)
Comment (emanuelschuetze) [reply openslides_votecollector/views.py:541]: We need the whiespace to get a nice linebreak for the Engage keypad display: 

```
Remove from
list of speakers
```

File: openslides_votecollector/views.py (hunk 5 above)
Comment (emanuelschuetze) [reply openslides_votecollector/views.py:536]: see below.

**Comment by emanuelschuetze** [comment]
@normanjaeckel done.

**Comment by normanjaeckel** [comment]
I generally do not want to commit compiled translation files into the repo. For release we can add fresh compilation to the built package. But you can leave them trhere for the moment and we can discuss this thing later.

**Comment by emanuelschuetze** [comment]
It standard also for OpenSlides core to commit the mo files, see https://github.com/OpenSlides/OpenSlides/tree/master/openslides/locale/de/LC_MESSAGES
